test/audience-pedantic.robot

```diff
@@ -0,0 +1,27 @@
+*** Settings ***
```

Comment:
I would keep all (audience) tests in the same file, adding just the proper labels to the tests.
I wouldn't use the term "pedantic", "strict" is better.
For non-strict tests I would use the label "relaxed" or "permissive".
I see that strict and relaxed tests have very similar code; we can define a new keyword with the common code and call that from the two tests.